### ISSUE TITLE
How to set the papersize of PDF output in PHPReport?

### ISSUE THREAD

**Issue by Amamatthew** [vernes/PHPReport#13]
Sometimes there's a extra width with too many table columns we need to specify the papersize property for the PDF output.

**Comment by vernes** [comment]
Yes, proper way is to add those properties as configurable parameters, becouse currently it's set to A4 [source](https://github.com/vernes/PHPReport/blob/master/PHPReport.php#L214)

**Comment by vernes** [comment]
If you want to change it yourself in the code, you can add any value supported by phpexcel found [here](https://github.com/PHPOffice/PHPExcel/blob/7d1c140974a4988f8a9739d167335d24fb59955e/Classes/PHPExcel/Writer/PDF/Core.php#L71)

**Comment by Amamatthew** [comment]
YES. I pass the arguments to the __construct() later:  $R=new PHPReport(array("PaperSize"=>64));

    /**
     * Creates new report with some configuration parameters
     * @param array $config 
     */
    public function __construct($config=array())
	{
                $this->setConfig($config);
		if(!empty($this->_PaperSize)){
			$this->init($this->_PaperSize);
		}else{
			$this->init("default");
		}
	}
    /**
     * Initializes internal objects 
     */
    private function init($_PaperSize)
    {
        if($this->_template!='')
        {
            $this->loadTemplate();
        }
        else
        {
            $this->createTemplate($_PaperSize);
        }
    }

	/**
	 * Creates PHPExcel object and template for report 为报告创建PHPExcel对象及模板
	 */
	private function createTemplate()
	{
		$this->objPHPExcel = new PHPExcel();
		$this->objPHPExcel->setActiveSheetIndex(0);
		$this->objWorksheet = $this->objPHPExcel->getActiveSheet();
		//TODO: other parameters	
		$this->objWorksheet->getPageSetup()->setOrientation(PHPExcel_Worksheet_PageSetup::ORIENTATION_PORTRAIT);
		
		if("default"==$this->_PaperSize){
			$this->objWorksheet->getPageSetup()->setPaperSize(PHPExcel_Worksheet_PageSetup::PAPERSIZE_A4);
		}else{
			$this->objWorksheet->getPageSetup()->setPaperSize($this->_PaperSize);
		}
		$this->objWorksheet->getPageSetup()->setHorizontalCentered(true);
		$this->objWorksheet->getPageSetup()->setVerticalCentered(false);
                $this->_usingTemplate=false;
	}
